compiler/liftdestructors.nim

```diff
@@ -230,6 +230,14 @@ proc fillBodyObjT(c: var TLiftCtx; t: PType, body, x, y: PNode) =
     # for every field (dependent on dest.kind):
     #   `=` dest.field, src.field
     # =destroy(blob)
+    if ccgIntroducedPtr(c.g.config,
+         (var dummy = newSym(skTemp, getIdent(c.g.cache, lowerings.genPrefix), nextSymId c.idgen, c.fn, c.info); dummy.typ = y.typ; dummy), y.typ):
```

Comment:
Move this outside the `ccgIntroducedPtr` call.

Reply:
Done